lib/soapy_cake/response.rb

```diff
@@ -29,6 +29,15 @@ def to_xml
       (empty? ? [] : [body.to_s]).to_enum
     end
 
+    def total_rows
+      body.to_s.scan(%r{<row_count>(\d+)<\/row_count>}).flatten.first.to_i
```

Comment:
We should use `match` here so it stops when it found the first occurrence (performance!).